client/modules/cm-lang-ftml/src/grammars/ftml.ts

```diff
@@ -21,112 +19,67 @@ const data = {
   blk_map: lkup(
     blockEntries
       .filter(([, { head, body }]) => head === "map" && body === "none")
-      .flatMap(aliases)
+      .flatMap(aliasesFiltered),
+    { ignoreCase: true }
   ),
 
   blk_val: lkup(
     blockEntries
       .filter(([, { head, body }]) => head === "value" && body === "none")
-      .flatMap(aliases)
+      .flatMap(aliasesFiltered),
+    { ignoreCase: true }
   ),
 
-  // currently empty
-  // blk_valmap: lkup(
-  //   blockEntries
-  //     .filter(([, { head, body }]) => head === "value+map" && body === "none")
-  //     .flatMap(aliases)
-  // ),
+  blk_valmap: lkup(
+    blockEntries
+      .filter(([, { head, body }]) => head === "value+map" && body === "none")
+      .flatMap(aliasesFiltered),
+    { ignoreCase: true }
+  ),
 
   // currently empty
   // blk_el: lkup(
   //   blockEntries
   //     .filter(([, { head, body }]) => head === "none" && body === "elements")
-  //     .flatMap(aliases)
+  //     .flatMap(aliases),
+  //  { ignoreCase: true }
   // ),
 
   blk_map_el: lkup(
     blockEntries
       .filter(([, { head, body }]) => head === "map" && body === "elements")
-      .flatMap(aliases)
+      .flatMap(aliasesFiltered),
+    { ignoreCase: true }
   ),
 
   blk_val_el: lkup(
     blockEntries
       .filter(([, { head, body }]) => head === "value" && body === "elements")
-      .flatMap(aliases)
+      .flatMap(aliasesFiltered),
+    { ignoreCase: true }
   ),
 
   // currently empty
   // blk_valmap_el: lkup(
   //   blockEntries
   //     .filter(([, { head, body }]) => head === "value+map" && body === "elements")
-  //     .flatMap(aliases)
+  //     .flatMap(aliases),
+  //  { ignoreCase: true }
   // ),
```

Comment:
Commented code?

Reply:
The `lkup` function isn't intended to take an empty array, so I've commented out this particular combination until a block type fulfills it.

I could make the function work with an empty array, sure, but I've find this kinda useful to keep track of things. Almost certainly a block of this type will get added at some point and uncommenting this will be my verification that everything is working.